flake.nix

```diff
@@ -150,7 +165,11 @@
               { nativeBuildInputs = [ pkgs.nodePackages.prettier ]; } ''
               cd ${./.} && prettier --check . && touch $out
             '';
-          };
+          } // pkgs.lib.optionalAttrs (system == "x86_64-linux") (import ./tests/legacy-distro-packaging.nix {
```

Comment:
I'm limiting the testing to x86_64 linux at this point because i don't have an ARM machine with KVM available right now.

The tests should be simple to adapt to aarch64 (they would have to obtain arm wireguard packages to support that).